### PR TITLE
migrate rna-star

### PR DESCRIPTION
**Pull request by czhu**
- [x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

**Comment by tseemann** [comment]
@czhu can you please squash into 1 commit please?
and change the tests to `assert_match version.to_s, shell_output("#{bin}/xxxx --version"))` etc?
thank you

**Comment by czhu** [comment]
fixed the tests. 
took me quite some time to squash on my end because I didn't know I have to force push to git repo. 
I thought you could merge commits for a pull request on your end with GitHub?

**Comment by jonchang** [comment]
>  I thought you could merge commits for a pull request on your end with GitHub?

typically yes, but bottling with brew uses a rebase workflow and the github buttons don't work with that

File: Formula/rna-star.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class RnaStar < Formula
+  desc "RNA-seq aligner"
+  homepage "https://github.com/alexdobin/STAR"
+  # cite "https://doi.org/10.1093/bioinformatics/bts635"
```
Comment (sjackman) [inline Formula/rna-star.rb]:
```suggestion
  # cite Dobin_2012: "https://doi.org/10.1093/bioinformatics/bts635"
```
Check out…
```sh
brew cite --ruby 10.1093/bioinformatics/bts635
```

File: Formula/rna-star.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  url "https://github.com/alexdobin/STAR/archive/2.7.3a.tar.gz"
+  version "2.7.3a"
+  sha256 "de204175351dc5f5ecc40cf458f224617654bdb8e00df55f0bb03a5727bf26f9"
+  head "https://github.com/alexdobin/STAR.git"
+
+  # Fix error: 'omp.h' file not found
+  depends_on "gcc"
```
Comment (sjackman) [inline Formula/rna-star.rb]: Is this line needed? I thought that `libomp` provided `omp.h` on macOS? And `gcc` is the default on Linux.

File: Formula/rna-star.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  depends_on "libomp" if OS.mac? # openMP
+
+  def install
+    ENV.cxx11
```
Comment (sjackman) [inline Formula/rna-star.rb]:
```suggestion
```
C++11 is the default now.

File: Formula/rna-star.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    if OS.mac?
+      ENV["CXX"] = Formula["gcc"].opt_bin/"g++-#{Formula["gcc"].version_suffix}"
+    end
```
Comment (sjackman) [inline Formula/rna-star.rb]: Does it work without these three lines?

**Comment by sjackman** [comment]
I just realized that I already migrated STAR, but I called the formula `star-aligner` and forgot about this old `rna-star` formula. See PR https://github.com/brewsci/homebrew-bio/pull/769.

**Comment by sjackman** [comment]
Thanks all the same, @czhu !

File: Formula/rna-star.rb (hunk 1 above)
Comment (czhu) [reply Formula/rna-star.rb]: "Unknown command: cite". where to get the cite command?

Comment (sjackman) [reply Formula/rna-star.rb]: `brew tap brewsci/base`

**Pull request closed without merging**